fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -1356,7 +1357,7 @@ On a high level, the API can be used to perform the following actions:
 |/transactionRequests|1.1|The data model is updated to add an optional ExtensionList element to the PartyIdInfo complex type based on the Change Request: https://github.com/mojaloop/mojaloop-specification/issues/30. Following this, the data model as specified in Table 93 has been updated.|
 |/quotes|1.1|The data model is updated to add an optional ExtensionList element to the PartyIdInfo complex type based on the Change Request: https://github.com/mojaloop/mojaloop-specification/issues/30. Following this, the data model as specified in Table 93 has been updated.|
 |/authorizations|1.0|The data model is updated to add an optional ExtensionList element to the PartyIdInfo complex type based on the Change Request: https://github.com/mojaloop/mojaloop-specification/issues/30. Following this, the data model as specified in Table 93 has been updated.|
-|/transfers|1.1|Added possible commit notification using PATCH /transfers/<ID>. The process of using commit notifications is described in Section 6.7.2.6. The data model is updated to add an optional ExtensionList element to the PartyIdInfo complex type based on the Change Request: https://github.com/mojaloop/mojaloop-specification/issues/30. Following this, the data model as specified in Table 93 has been updated.|
+|/transfers|1.2|Added a new method of notification using HTTP PATCH on resource /transfers/{ID}/error, when there is a validation or another processing error (not schema validation) during the processing of a PUT /transfers/{ID} sent from the Payee FSP, both when RESERVED and COMMITTED is used as transferState. //The process of using commit notifications is described in Section 6.7.2.6//. This is based on the Change Request: https://github.com/mojaloop/mojaloop-specification/issues/103.|
```

Comment:
The **PATCH /transfers/**_{ID}_**/error** notification must not be sent in version 1.2 when `transferState` `RESERVED` is used, see https://github.com/mojaloop/mojaloop-specification/issues/103#issuecomment-970407712.

The following should be removed as it was introduced in version 1.1:

> //The process of using commit notifications is described in Section 6.7.2.6//

Reply:
Fixed - this applies only for the transferState COMMITTED (for 1.2)